From 4e8ba066a5e6fbb41a4a4d01a3baef8b0eb23caf Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 22 Aug 2019 22:14:14 +0200
Subject: [PATCH 1/1] cmd: gpio: remove redundant assignment

The assigned value NULL is overwritten before being used. Remove the
assignment.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/gpio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/gpio.c b/cmd/gpio.c
index 53366f36e7..eff36ab2af 100644
--- a/cmd/gpio.c
+++ b/cmd/gpio.c
@@ -91,7 +91,7 @@ static int do_gpio_status(bool all, const char *gpio_name)
 
 		if (!gpio_name || !bank_name ||
 		    !strncasecmp(gpio_name, bank_name, banklen)) {
-			const char *p = NULL;
+			const char *p;
 			int offset;
 
 			p = gpio_name + banklen;
-- 
2.20.1

